src/pymodaq_plugins_newport/daq_move_plugins/daq_move_Newport_XPS_Q8.py

```diff
@@ -0,0 +1,294 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, comon_parameters_fun, main, DataActuatorType,\
+    DataActuator  # common set of parameters for all actuators
+from pymodaq.utils.daq_utils import ThreadCommand # object used to send info back to the main thread
+from pymodaq.utils.parameter import Parameter
+from qtpy.QtCore import QThread
+from pymodaq_plugins_newport.hardware import XPS_Q8_drivers 
+import sys
+
+from time import perf_counter_ns
+
+class XPSPythonWrapper():
+    """Simplified XPS wrapper, calls methods from the wrapper given by Newport. See XPS_Q8_drivers"""
+    
+    def __init__(self, ip:str = None, port:int = None, group:str = None, positionner:str = None, plugin = None):
+        #init the wrapper given by Newport and some attributes
```

Comment:
please add docstring precising what should be your paramaters

Reply:
done on commit [32d9cd8](https://github.com/PyMoDAQ/pymodaq_plugins_newport/pull/6/commits/32d9cd836e5e27d17f38f65f06bc1b10124783bd)